Application/Frontend/components/Server/Member_List.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { useEffect, useState } from 'react';
 import { Avatar, Group, Text, Stack, Paper, Button, Menu, rem } from '@mantine/core';
-import { IconSettings, IconMessageCircle, IconPhoto, IconSearch, IconArrowsLeftRight, IconTrash } from '@tabler/icons-react';
+import { IconSettings, IconMessageCircle, IconPhoto, IconSearch, IconArrowsLeftRight, IconTrash, IconPlus, IconUserUp } from '@tabler/icons-react';
```

Comment:
Looks like you have picked up a few more Icons import from our React library - welcomed changes